interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -681,8 +681,8 @@ namespace cling {
     cling::valuePrinterInternal::declarePrintValue(*this);
     std::string ret;
     std::stringstream ss;
-    ss << "*((std::string*)" << &ret << ") = cling::printValue((" << type << "*)"
-       << obj << ");";
+    ss << "*((std::string*)" << std::hex << std::showbase << (size_t)&ret << ") = cling::printValue((" << type << "*)"
```

Comment:
Can we use [`llvm::raw_string_ostream`](http://llvm.org/doxygen/classllvm_1_1raw__string__ostream.html) instead of `std::stringstream`? That's properly platform independent.

Reply:
No idea, I'll try

Reply:
Thanks for the suggestion!